src/types.ts

```diff
@@ -24,6 +24,8 @@ export type DataMigrationOptions<TSchema> = {
   maxConcurrentUpdateCalls: number;
   /** Restricted projection which will be backed up - use this if you need all `projection` keys to compute the update, but you are editing a subset */
   projectionBackupFilter?: Array<keyof TSchema>;
+  /** Keep the automatic rollback logic */
+  rollbackable: boolean;
```

Comment:
@JulienBara-360 I don't understand the doc meaning
Could be more helpful with _"Won't backup document and will silently do nothing at rollback procedure"_ for instance